lib/screens/second_page.dart

```diff
@@ -0,0 +1,77 @@
+import 'package:flutter/material.dart';
+
+class SecondPage extends StatelessWidget {
```

Comment:
SecondPage is not the best name, it does not says anything about the content, how about DetailsPage?

Reply:
Done